debian/control

```diff
@@ -56,3 +56,10 @@ Package: cloudstack-integration-tests
 Architecture: all
 Depends: ${misc:Depends}, cloudstack-marvin (= ${source:Version})
 Description: The CloudStack Marvin integration tests
+
+Package: cloudstack-agent-ha-helper
```

Comment:
@GabrielBrascher we'll need similar for CentOS7, 8 and Suse packaging.
Can you advise why we need a separate process, can't the agent do this or a thread in libvirtcomputingresource? And it seems it may create a security issue, is the service port (8080) secured and authenticated?